ppl/cmd/zqd/listen/command.go

```diff
@@ -88,6 +88,8 @@ func New(parent charm.Command, f *flag.FlagSet) (charm.Command, error) {
 	f.StringVar(&c.suricataUpdaterPath, "suricataupdater", "", "command to update Suricata rules (run once at startup)")
 	f.StringVar(&c.workers, "workers", "", "workers as comma-separated [addr]:port list")
 	f.StringVar(&c.zeekRunnerPath, "zeekrunner", "", "command to generate Zeek logs from pcap data")
+
+	c.conf.Auth.SetFlags(f)
```

Comment:
Nit: Move this up before `c.conf.Version = cli.Version` to consolidate c.conf twiddling.